src/main/java/me/zeroeightsix/kami/module/modules/movement/AutoWalk.kt

```diff
@@ -113,11 +126,23 @@ class AutoWalk : Module() {
     }
 
     enum class AutoWalkMode {
-        FORWARD, BACKWARDS, BARITONE
+        FORWARD, BACKWARDS, PATHFIND
     }
 
     companion object {
         @JvmField
         var direction: String? = null
     }
+
+    override fun onWorldRender(event: RenderEvent?) {
+        if (drawPath.value && mode.value.equals(AutoWalkMode.PATHFIND) && MODULE_MANAGER.isModuleEnabled(ElytraFlight::class.java) && Astarpathfinder.path != null) {
+            val lines: MutableList<Vec3d> = mutableListOf()
+            for (i in 0 until Astarpathfinder.path.size) {
+                lines.add(Vec3d(Astarpathfinder.path[i].pos.x + 0.5, mc.player.posY - 0.5, Astarpathfinder.path[i].pos.z + 0.5))
+            }
+            GlStateManager.enableTexture2D()
+            KamiTessellator.drawLinesFromList(lines, 1f, 1f, 0f, 1f)
+            KamiTessellator.releaseGL()
+        }
+    }
```

Comment:
I'll probably rewrite this GL myself before merging

Reply:
feel free :)